lang/LangPrimSource/PyrStringPrim.cpp

```diff
@@ -558,64 +557,63 @@ int prString_PathMatch(struct VMGlobals* g, int numArgsPushed) {
 }
 
 int prString_Getenv(struct VMGlobals* g, int /* numArgsPushed */) {
-    PyrSlot* arg = g->sp;
-    char key[256];
-    char* value;
-    int err;
-
-    err = slotStrVal(arg, key, 256);
-    if (err)
-        return err;
+    PyrSlot* slot_this = g->sp;
+    const auto [this_err, this_str] = slotStdStrVal(slot_this);
+    if (this_err != errNone)
+        return this_err;
 
 #ifdef _WIN32
+    // TODO: windows is limited to 1024, is this necessary?
```

Comment:
See documentation: https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getenvironmentvariable

You can check return value from passing in size 0

Reply:
Thanks, I think I've fixed this now. 

By the way, is there any reason the project is on clang-format-8 rather than a newer version? Took me ages to find that issue!